Landry333/Big-Owl
# #30 Set Schedule

### Important Note (as of January 14)
Right now, github is updating their CI machines and its leading to build issues. As a result, Reviewdog builds will be failing for now. Here's a link explaining the issue if you are curious:
https://github.com/actions/virtual-environments/issues/2420

----


### Related Issues
#30, #178 (just a tiny bug fix)

### Description
Added a page where schedules can be added to selected members of a group.

### How to check PR

1. Login as a user with a group that has a couple of users (ex: bigowl123)
2. Click on SetSchedule button
3. Fill out the form
4. Click on Set Schedule Button to add the schedule

Result: You should return to the previous page and there should be a new schedule in the database in the schedules collection if Firestore

###  Important Files:

- SetScheduleActivity.java
- Schedule.java
- SetScheduleViewModel.java

### Images
![image](https://user-images.githubusercontent.com/31963426/104114107-89964780-52ce-11eb-9843-f7fcb514848d.png)
![image](https://user-images.githubusercontent.com/31963426/104114108-931faf80-52ce-11eb-9d58-225aa1eb7f6f.png)
